src/components/common/Header/Header.tsx

```diff
@@ -0,0 +1,13 @@
+const Header: React.FC = () => {
+  return (
+    <header className="bg-primary h-16">
+      <div className="flex items-center justify-left h-full">
+        <a className="text-white text-5xl ">
+          Chosei-Chan
+        </a>
+      </div>
+    </header>
```

Comment:
`Chosei-Chan`を中央に寄せたいだけであればこのdivはいらないかな〜
あと、Linkでどこかに飛ばしたいとかではなければpタグで大丈夫
ロゴ押したらトップページとかに戻るのであれば`href="/" target="_blank" rel="noopener noreferrer"`を追加しよう
参考：https://webst8.com/blog/rel-noopener-noreferrer/

後もうちょっと横のpaddingを増やして、文字を小さくしてもいいかな（キツキツになってる）
（横のpaddingはpx-〇〇でつけれるよ）

```suggestion
    <header className="flex items-center bg-primary h-16">
      <a className="text-white text-4xl" href="/" target="_blank" rel="noopener noreferrer">
        Chosei-Chan
      </a>
    </header>
```